internal/api/server.go

```diff
@@ -26,13 +27,15 @@ import (
 	"github.com/akuity/kargo/internal/api/option"
 	"github.com/akuity/kargo/internal/api/validation"
 	rollouts "github.com/akuity/kargo/internal/controller/rollouts/api/v1alpha1"
-	httputil "github.com/akuity/kargo/internal/http"
 	"github.com/akuity/kargo/internal/logging"
 	"github.com/akuity/kargo/pkg/api/service/v1alpha1/svcv1alpha1connect"
 )
 
 var (
 	_ svcv1alpha1connect.KargoServiceHandler = &server{}
+
+	//go:embed all:ui
+	ui embed.FS
```

Comment:
Note that this is the reason for the empty `internal/api/ui/.keep` file. It is necessary to avoid compilation errors.